blog/2022-07-27-cwa-2-25/index.md

```diff
@@ -7,7 +7,7 @@ author: CWA-Team, 6 pm
 layout: blog
 ---
 
-The project team of the Robert Koch-Institute (RKI), Deutsche Telekom, and SAP have **released version 2.25 of the Corona-Warn-App (CWA).** The team has updated the **guidelines on what to do in case of a low or increased risk** (red as well as green tile) correspondent to the [recommendations of the RKI](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Quarantaene/Absonderung.html;jsessionid=EE3180A1446820CBF8BFC7BC793F763A.internet062?nn=13490888) (German only). Among other things, this involves recommendations on contact restriction and wearing masks. In addition, the project team has made improvements and small adjustments. 
+The project team of the Robert Koch-Institute (RKI), Deutsche Telekom, and SAP have **released version 2.25 of the Corona-Warn-App (CWA).** The team has updated the **guidelines on what to do in case of a low or increased risk** (red as well as green tile) correspondent to the [recommendations of the RKI](https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Quarantaene/Absonderung.html) (German only). Among other things, this involves recommendations on contact restriction and wearing masks. In addition, the project team has made improvements and small adjustments. 
```

Comment:
Also "correspondent" should be "corresponding".

https://dictionary.cambridge.org/dictionary/english/correspondent is a noun.

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/3057/commits/178ffceb9817be09010aee60c4e74b76763972b3, thanks!